config/manager/kustomization.yaml

```diff
@@ -4,5 +4,5 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 images:
 - name: controller
-  newName: summerwind/actions-runner-controller
+  newName: ghcr.io/cory-miller/actions-runner-controller
```

Comment:
NIT: newName

Reply:
On further inspection this is the spec field for an image override (https://pkg.go.dev/github.com/fluxcd/pkg/apis/kustomize#Image). I reverted the name though.